core/api/v1/keytransparency_proto/keytransparency.proto

```diff
@@ -217,7 +217,26 @@ message Epoch {
   repeated bytes log_consistency = 4;
   // log_inclusion proves that smr is part of log_root at index=srm.MapRevision.
   repeated bytes log_inclusion = 5;
-  // mutation contains mutation information.
+}
+
+// ListMutationsRequest contains the input parameters of the GetMutation APIs.
+message ListMutationsRequest {
+  // domain_id is the domain for which epochs are being requested.
+  string domain_id = 5;
+  // epoch specifies the epoch number in which mutations will be returned.
+  int64 epoch = 1;
+  // page_token defines the starting point for pagination. An empty
+  // value means start from the beginning. A non-empty value requests the next
```

Comment:
if all there is to this is empty/non-empty then it could be bool!

The non-empty value must be coming from somewhere, and must encoding the particular page start point, I'd imagine?

Reply:
The idea is that `page_token` should be opaque to the client. 
Fetching the next page involves passing the `next_page_token` from the previous response. 
To start from zero, don't send `page_token`.

Reply:
Updated the documentation. To fetch the next page, the client supplies the`next_page_token` value from the previous response.